### ISSUE TITLE
Question on BatchNorm

### ISSUE THREAD

**Issue by breakds** [pkmital/tensorflow_tutorials#8]
I read the code in batch_norm, and found that self.ema (the ExponentialMovingAverage object) is never used. I thought at some point we need to call self.ema.apply([self.mean, slef.variance]) to create the updater of the moving average? The moving average will be used in validation/testing stage, instead of the batch mean/variance, I think.

**Comment by pkmital** [comment]
Thanks!  I've just updated the 06_modern_convnet example and the batch norm implementation to address this.

**Comment by breakds** [comment]
Thanks a lot, Parag!
